### ISSUE TITLE
Accessibility: Content not within landmark regions (ARIA) violates "2.4.1 Bypass Blocks" Requirement

### ISSUE THREAD

**Issue by anitakumari28** [microsoft/vscode#250108]
- VS Code Version: 1.100.2
- OS Version: macOS Sequoia 15.4.1

Content on the extension’s webview page is not enclosed within any ARIA landmark regions such as ```<main>, <nav>,``` or containers with role="main", role="region", etc.
This causes accessibility issues ([Understanding 2.4.1 Bypass Blocks](https://www.w3.org/WAI/WCAG22/Understanding/bypass-blocks))for users relying on assistive technologies like screen readers, which use landmarks to navigate between sections of content.

Steps to Reproduce:

1. Open the WCA4Z extension webview in VS Code.
2. Elements present in the references are not within a landmark.

Expected behavior
 All meaningful content should be enclosed within appropriate landmark containers to support semantic structure and keyboard/screen reader navigation. For example:
```html
<main role="main">
  <!-- Primary content here -->
</main>
```
Affected Element (xPath):

1. /html/body/p[1]  
   → "Select a file or folder from the COBOL that is ready for transformation..."

2. /html/body/div[@class='button-container']  
   → <a class="monaco-button monaco-text-button"><span>Select COBOL</span></a>

3. /html/body/h3[@class='title']  
   → Heading: "IBM watsonx Code Assistant for Z" or equivalent```

References:

1. 
 <img width="500" alt="Image" src="https://github.com/user-attachments/assets/6e674643-2db1-4af7-8f55-39ccf884bd21" />
2.
 <img width="500" alt="Image" src="https://github.com/user-attachments/assets/12147fb9-6e84-4289-88b5-08a6f7fbedc2" />
3.
 <img width="500" alt="Image" src="https://github.com/user-attachments/assets/1c656924-7849-4794-95ec-5bd4f2e2ec19" />

**Comment by sandy081** [comment]
cc @bhavyaus owns welcome views

**Comment by meganrogge** [comment]
No users have reported this, so not urgent to fix.

**Comment by sandy081** [comment]
> That is controlled by VS Code but does not appear to be a webview. @sandy081 Can you please take a look at that part to see if we're doing accessibility incorrect there

This seems to be a welcome content in contributed view. Hence assigning to @benibenj - I guess you are owning this now (before @joaomoreno)
